static/vis_D3.css

```
@@ -0,0 +1,128 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font: 16px Arial;
```

```suggestion
  font-size: 16px;
  font-family: Lato, Helvetica Neue, Arial, sans-serif;
```
Let's use the same font-family as sphinx-rtd-theme.